DataStructures/HashTable/HashTable.cs

```diff
@@ -1,18 +1,23 @@
 ﻿using System;
+using System.ComponentModel;
 using System.Data;
 
 
 namespace DataStructures.HashTable
 {
     public class HashTable : IHashTable
     {
-        private object[] _array;
+        private class ArrayValue
```

Comment:
Please take a look where Node<> class is in LinkedList (at the end of the file) and do the same with ArrayValue class.

Reply:
Understand